download/download.html.haml

```diff
@@ -101,64 +101,6 @@ title: Download
       * For Red Hat subscription releases https://access.redhat.com/downloads[go to the product download site].
 
 :asciidoc
-  [[NonFinalReleases]]
```

Comment:
We don't want to remove these sections.

Either we just leave them in (which I like)
or we do a haml if check to only show them when latestFinal and latest are different (which I can live with)